data/json/items/vehicle/armor.json

```
@@ -20,7 +20,7 @@
     "type": "GENERIC",
     "category": "veh_parts",
     "name": { "str": "shock absorber" },
-    "description": "This makeshift combination of springs and scrap, when attached to a vehicle section, protects that section from impacts.  The springs can absorb a surprising amount of damage.",
+    "description": "A makeshift combination of springs and scrap that protects a particular section of a vehicle from impacts.  The springs can absorb a surprising amount of damage.",
```

```suggestion
    "description": "A makeshift combination of springs and scrap that protects a particular section of a vehicle from indirect damage from impacts.  The springs can absorb a surprising amount of damage.",
```
There's probably a better way to say this but I can't think of it right now.